scripts/skippedTestReport.ts

```diff
@@ -0,0 +1,113 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Scans test files for skipped Mocha tests.
+ *
+ * It uses a regex instead of mocha's loader, because mocha's
+ * loader can't resolve vscode by default
+ */
+
+import * as fs from 'fs'
+import * as path from 'path'
+
+interface SkippedTest {
+    file: string
+    testName: string
+    lineNumber: number
+}
+
+function findSkippedTests(directoryPath: string): SkippedTest[] {
+    const skippedTests: SkippedTest[] = []
+
+    const skipPatterns = [/\b(describe|it)\.skip\(['"`](.*?)['"`]/g]
```

Comment:
What about something like https://github.com/aws/aws-toolkit-vscode/blob/29c4048dce1759713af0a9b2637538e37c237a63/packages/core/src/test/codecatalyst/uriHandlers.test.ts#L122-L137

I can't think of a good solution here since it can be conditionally skipped or always skipped via `this.skip`. Do you think its worth including these as a separate "conditionally skipped"?